README.md

```diff
@@ -1822,6 +1822,22 @@ get do
 end
 ```
 
+The above example may have been requested as follows:
+
+``` shell
+curl -H "secret_PassWord: swordfish" ...
+```
+
+The header name will have been normalized for you.
+
+- In the `header` helper names will be coerced into a capitalized kebab case.
+- In the `env` collection they appear in all uppercase, in snake case, and prefixed with 'HTTP_'.
+
+With this normalization you will be able to handle headers in a case-insensitive way as per
```

Comment:
For an API documentation we don't need to convince people this is a good idea. I would put this in the prefix. 

_The header name will have been normalized per HTTP standards defined in ... regardless of what is being sent by a client._

(fix HTTP type, 2 Ts)

Reply:
Updates committed.